app/public/css/styles.css

```diff
@@ -478,6 +482,75 @@ p {
   fill: #fff;
 }
 
+.button.undelegate {
+  border: 1px solid var(--color-grey);
+  font-size: 14px;
+  font-weight: 400;
+  background-color: transparent;
+  padding: 0px 4px;
+  color: var(--color-grey);
+  white-space: nowrap;
+}
```

Comment:
regarding all css pixel numbers, we try to follow multiples-of-4 convention, unless it's absolutely needed otherwise. That applies to all other css changes. Also, this button is way too small. Please give it static width and height so that it looks more like other buttons concerning size.

Reply:
- Does the multiples-of-4 convention apply to font-size? There are multiple instances of it not being the case.

- I changed the padding of the delegate/undelegate buttons to `6px 10px` to match the donations buttons. Open to discussion as they're not multiples of 4.

Reply:
font size - not necessarily, regarding that, we try to not have too many different font sizes, so that its similar across the page